doc/interfaces/data_fetch.rst

```diff
@@ -0,0 +1,22 @@
+------------------------
+Downloading the Dataset
+------------------------
+In this tutorial, you will learn to download dataset using the terminal.
+
+First, let’s create a folder::
+
+    mkdir data_folder
+
+You can view all the datasets using the command::
+
+    dipy_fetch list
+
+Download the data in the ``data_folder``::
```

Comment:
In order to fetch a specific dataset, you will specify its name with the {double_backticks}dipy_fetch{double_backticks} command, and an optional destination directory using the {double_backticks}--out_dir{double_backticks} argument::
    dipy_fetch {specific_dataset} --out_dir {data_folder}

To me, using the braces or some other hint (always the same across all CLIs) allows the user to realized that they need to substitute them by the appropriate values.

Reply:
Addressed in 851afcb